Monika After Story/game/definitions.rpy

```diff
@@ -2959,6 +2959,61 @@ init -1 python:
     def mas_isMonikaBirthday():
         return datetime.date.today() == mas_monika_birthday
 
+    def mas_isSpring():
```

Comment:
can you make this (and the other functions) more like the `mas_isO31` function where it accepts a `_date` parameter, defaulted to today?